src/server.c

```diff
@@ -133,23 +157,48 @@ void serverLogRaw(int level, const char *msg) {
     } else {
         int off;
         struct timeval tv;
-        int role_char;
         pid_t pid = getpid();
         int daylight_active = atomic_load_explicit(&server.daylight_active, memory_order_relaxed);
 
         gettimeofday(&tv, NULL);
         struct tm tm;
         nolocks_localtime(&tm, tv.tv_sec, server.timezone, daylight_active);
-        off = strftime(buf, sizeof(buf), "%d %b %Y %H:%M:%S.", &tm);
-        snprintf(buf + off, sizeof(buf) - off, "%03d", (int)tv.tv_usec / 1000);
+        switch (server.log_timestamp_format) {
+        case LOG_TIMESTAMP_DEFAULT:
+            off = strftime(buf, sizeof(buf), "%d %b %Y %H:%M:%S.", &tm);
+            snprintf(buf + off, sizeof(buf) - off, "%03d", (int)tv.tv_usec / 1000);
+            break;
+
+        case LOG_TIMESTAMP_ISO8601:
+            off = strftime(buf, sizeof(buf), "%Y-%m-%dT%H:%M:%S.", &tm);
+            char tzbuf[7];
+            format_timezone(tzbuf, server.timezone, server.daylight_active);
```

Comment:
```suggestion
            char tzbuf[7];
            format_timezone(tzbuf, sizeof(tzbuf), server.timezone, server.daylight_active);
```
Based of my other comment.

Reply:
Is it already possible to configure the timezone? To log everything in UTC, the user can just set server timezone to UTC?

Reply:
Yeah, it is possible, but I think some people prefer to leave the timezone enabled on their machine?